browse-scaler/src/cmr.js

```diff
@@ -4,6 +4,7 @@ const fetch = require('node-fetch');
 const cmrRootUrl = `http://${process.env.CMR_ROOT}`;
 const cmrCollectionUrl = `${cmrRootUrl}/search/collections.json?concept_id=`;
 const cmrGranuleUrl = `${cmrRootUrl}/search/granules.json?concept_id=`;
+const echoToken = process.env.ECHO_TOKEN || '';
```

Comment:
We should probably fetch this from parameter store rather than passing in an environment variable.